packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/order/choice/telephony-line-phone-order-choice.html

```diff
@@ -0,0 +1,180 @@
+                                            alt="{{phone.description}}"
+                                            data-ng-src="{{phone.img}}"
+                                        />
+                                        <span
+                                            data-ng-if="!phone.img"
+                                            class="text-center ovh-font ovh-font-phone"
+                                        ></span>
+                                    </div>
+                                    <div class="caption">
+                                        <div class="text-center">
+                                            <span
+                                                data-translate="telephony_line_phone_deposit"
+                                            ></span>
+                                            <span
+                                                data-ng-bind="phone.price.text"
+                                            ></span>
+                                        </div>
+                                        <div class="text-center oui-label">
+                                            <span
+                                                data-ng-bind="phone.setupFees.description"
+                                            ></span>
+                                            <span
+                                                data-ng-bind="phone.setupFees.price.text"
+                                            ></span>
+                                        </div>
+                                        <div
+                                            class="text-center oui-label"
+                                            data-ng-if="phone.url"
+                                        >
+                                            <a
+                                                data-ng-href="{{phone.url}}"
+                                                target="_blank"
+                                                rel="noopener"
+                                            >
+                                                <span
+                                                    data-translate="telephony_line_phone_detail_link"
+                                                >
+                                                </span>
+                                            </a>
+                                        </div>
+                                    </div>
+                                </oui-radio>
+                            </oui-radio-group>
+                        </div>
+                    </div>
+                </oui-tabs-item>
+            </oui-tabs>
+            <div
+                class="alert alert-warning alert-xs mt-3"
+                role="alert"
+                data-ng-if="$ctrl.isSamePhone()"
+            >
+                <i class="ovh-font ovh-font-filled-warning" aria-hidden="true">
+                </i>
```

Comment:
Please use `<span>` for images instead, as `<i>` tag is meant to be used to display a specific term